apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -146,6 +141,14 @@ export default function CourseMap() {
           }
         : undefined;
 
+    const hasWeekendCourse = AppStore.getCourseEventsInCalendar().some(
+        (event) => event.start.getDay() === 0 || event.start.getDay() === 6
+    );
```

Comment:
Also, just a nitpick but this (and a lot of the other consts in this component) should be memoized. e.g.
```ts
    const hasWeekendCourse = useMemo(() => AppStore.getCourseEventsInCalendar().some(
        (event) => event.start.getDay() === 0 || event.start.getDay() === 6
    ), [AppStore.courseEventsInCalendar])
```
I'm not actually sure if this works properly because the store isn't a conventional observable. But that's the idea at least.